.changeset/lemon-ducks-breathe.md

```diff
@@ -0,0 +1,8 @@
+---
+'@remirror/extension-annotation': minor
+'@remirror/storybook': minor
```

Comment:
It doesn’t like the fact that you’ve added the `@remirror/storybook` package which is ignored by change sets. That’s the reason it’s failing. 

```suggestion
```